README.md

```diff
@@ -92,7 +94,8 @@ This list contains tools, books, articles, blogs, courses and everything related
 - [Polyglot](https://github.com/untra/polyglot) - A fast, painless, open-source internationalization plugin for Jekyll blogs.
 - [ICU - International Components for Unicode](https://icu.unicode.org/) - A mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization support for software applications.
 - [Django docs - i18n](https://docs.djangoproject.com/en/4.1/topics/i18n/) - Part of the Django docs, focused on internationalization and localization.
-  [Localizejs](https://localizejs.com/) - Translations for websites and applications to new languages and streamline translation workflow.
+- [Localizejs](https://localizejs.com/) - Translations for websites and applications to new languages and streamline translation workflow.
```

Comment:
Looks like added free space:
![free](https://github.com/mbiesiad/awesome-translations/assets/18367606/86e121c7-19d9-469e-b218-0d5dee65cf27)
Please remove redundant free space (lines) there.

Reply:
I think I just fixed the list. See original:
![image](https://github.com/mbiesiad/awesome-translations/assets/58360188/06f51fd6-4dbb-4b67-acc9-c9dfb70cc731)

Reply:
Wow, my bad. You're right! 🔥 Thank you so much!